llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -137,6 +137,13 @@ static void dumpLLVM(Module *m, const std::string &fName) {
   }
 }
 
+static bool isStorageClassExplicitlyLaidOut(SPIRVModule *m_bm, SPIRVStorageClassKind storageClass) {
+
```

Comment:
Thanks for outlining this, I think it makes the callsites much cleaner.

nit: no need for an empty line before `return`